askama_derive/src/generator.rs

```diff
@@ -2033,6 +2038,14 @@ fn median(sizes: &mut [usize]) -> usize {
     }
 }
 
+fn insert_variable_at_the_end(args: &mut Vec<Expr<'_>>, var_name: String) {
```

Comment:
It's a bit counter-intuitive because the first filter called is actually the one at the "bottom". I'll rename and add documentation to clarify all this.